nexus/tests/output/nexus_tags.txt

```diff
@@ -4,6 +4,7 @@ disk_create                              /organizations/{organization_name}/proj
 disk_delete                              /organizations/{organization_name}/projects/{project_name}/disks/{disk_name}
 disk_list                                /organizations/{organization_name}/projects/{project_name}/disks
 disk_view                                /organizations/{organization_name}/projects/{project_name}/disks/{disk_name}
+disk_view_by_id                          /by-id/disks/{id}
```

Comment:
What do you think about flipping the paths around like this:

```suggestion
disk_view_by_id                         /disks/by_id/{id}
```

It follows path convention a little closer and it would make command generation a little easier.

Reply:
Would the idea be that you'd have `/disks`, `/projects`, `/instances`, and all the others at the top level, and each of these would contain only one HTTP resource beneath them which was `by_id`?

Reply:
I'm not sure if we are planning to have more global actions on resources? For example, we might want to have batch actions in the future where several resources are acted on.

In that case having the `by_id` bit of the path wouldn't make too much sense any more, the way I'm proposing 🤔 

Also, we'll probably want to have an endpoint for attaching disks with an ID as well. That would make it `instances/{id}/disks/attach/{id}`

I'm OK with either way we go :) If the majority prefer having `by_id` at the top level I'm ok with that :)

Reply:
As for batching, I think the current endpoint could still support that. Something like `/by-id/{resource}?ids[]=1,2,3,4` is a rough approximation of how it could work.

Reply:
Ha! I had not seen https://github.com/oxidecomputer/omicron/issues/1383. Sounds good to me then :)